dev/breeze/src/airflow_breeze/utils/run_utils.py

```diff
@@ -40,12 +41,22 @@ def run_command(
     verbose: bool = False,
     suppress_raise_exception: bool = False,
     suppress_console_print: bool = False,
+    env: Optional[Mapping[str, str]] = None,
     **kwargs,
 ):
     if verbose:
-        console.print(f"[blue]$ {' '.join(shlex.quote(c) for c in cmd)}")
+        command_to_print = ' '.join(shlex.quote(c) for c in cmd)
```

Comment:
And Also it allows to update the "verbose" part of run_command - instead of printing the variables and commands outside - I can now also print all the environment variables here - which means that it will work in the future as well in any place where you pass env parameter to run_command.